docs/reference/docs/delete.asciidoc

```diff
@@ -105,7 +105,8 @@ thrown instead.
 [[delete-index-creation]]
 === Automatic index creation
 
-The delete operation automatically creates an index if it has not been
+If external version is used,
```

Comment:
What about adding a link here as there are actually 3 external version types : `external`, `external_gt` and `external_gte` ?